CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Gravity is an open source project.
+
+It is the work of tens of contributors. We appricate your help!
+
+We follow a [code of conduct](./CODE_OF_CONDUCT.md).
+
+
+## Filing an Issue
+
+Security issues should be reported directly to security@gravitational.com.
+
+If you are unsure if you've found a bug, consider searching
+[current issues](https://github.com/gravitational/gravity/issues) or
+asking in the [community forum](https://community.gravitational.com/) first.
+
+Once you know you have a issue, make sure to fill out all sections of the
+one of the templates at https://github.com/gravitational/gravity/issues/new/choose.
+
+Gravity contributors will triage the issue shortly.
+
+
+## Contributing A Patch
+
+If you're working on an existing issue, respond to the issue and express
+interest in working on it.  This helps other people know that the issue is
```

Comment:
Nit: Extra space between the dot and "This".

Reply:
addressed by 1c5ffc3